### PR TITLE
Add documentation for version-checks in the editor

### PR DESCRIPTION
**Pull request by peyerluk**

### CODE REVIEW

File: reference-docs/editor-configuration/version-checks.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+work with different editor versions.
+
+
+#### Requirement #1 - version.json file present
+
+A version.json file must be generated when deploying the editor. This file
+must be in the same folder as index.html.
+
+The format of the version.json file is not defined. The editor will simply issue an
+update message as soon as the files contents change.
```
Comment (marcbachmann) [inline reference-docs/editor-configuration/version-checks.md]: A json object without comments is the defined format ;)

File: reference-docs/editor-configuration/version-checks.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Version checks allow to notify the user as soon as a new editor is released. This
+helps users upgrade to the latest editor quickly and reduces problems if users
+work with different editor versions.
+
+
+#### Requirement #1 - version.json file present
+
+A version.json file must be generated when deploying the editor. This file
+must be in the same folder as index.html.
```
Comment (marcbachmann) [inline reference-docs/editor-configuration/version-checks.md]: Please be more specific. Customers don't have an `index.html`

Comment (marcbachmann) [reply reference-docs/editor-configuration/version-checks.md]: In our build step we're creating a `dist` (this path is based on an environment variable).
The `version.json` file belongs into this directory.

**Comment by peyerluk** [comment]
@marcbachmann I updated the description where the verrsion.json belongs according to you comment.

File: reference-docs/editor-configuration/version-checks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+## Version-Checks
+
+Version checks allow to notify the user as soon as a new editor is released. This
+helps users upgrade to the latest editor quickly and reduces problems if users
+work with different editor versions.
+
+
+#### Requirement #1 - version.json file present
+
+A version.json file must be generated when deploying the editor. This file must be in the dist directory created in the build step (the path is based on an environment variable).
```
Comment (marcbachmann) [inline reference-docs/editor-configuration/version-checks.md]: Here we should link to some build documentation.
Sadly we don't have any build documentation for the editor at all.

**Pull request merged**